test/battle/battle.test.ts

```diff
@@ -182,7 +182,7 @@ describe("Test Battle Phase", () => {
     game.override.moveset([MoveId.TACKLE]);
     game.override.enemyAbility(AbilityId.HYDRATION);
     game.override.enemyMoveset([MoveId.TACKLE, MoveId.TACKLE, MoveId.TACKLE, MoveId.TACKLE]);
-    await game.classicMode.startBattle();
+    await game.classicMode.startBattle(SpeciesId.FEEBAS);
```

Comment:
```suggestion
    await game.classicMode.startBattle(SpeciesId.MEWTWO);
```
starterSpecies is overriden to Mewtwo line 178, so that line should be removed

We may want to take a closer look at the starter species override VS this `generateStarter` helper. If there are meaningful differences we should note them and make them clear in the documentation. If we always want to use startBattle rather than the override, maybe `startBattle` should fail or at least print a warning if there is a species override?